docs/accessing-services.asciidoc

```diff
@@ -0,0 +1,168 @@
+[id="{p}-accessing-elastic-services"]
+= How to access Elastic Stack services
+
+To access the Elastic Stack services, make sure you have:
+
+- certificate validation with the CA (Certificate Authority)
+- basic authentication (Elastic user password)
+- IP of the service
+
```

Comment:
I find this paragraph very confusing. Are we expecting people to have access to things they are just about to learn about in this document? Is this this meant as a prerequisite before continuing reading this help document (incorrect)? Is this meant as a summary of what is about to come below? If so this needs to be made clear.

Reply:
To my understanding, this is meant as a summary of what is about to come below. 

How about this:

```
To access the Elastic Stack services, you will need to retrieve:

- certificate validation with the CA (Certificate Authority)
- basic authentication (Elastic user password)
- IP of the service
```

Reply:
That is also not correct because you can access Elasticsearch without knowing the IP of the service from within the cluster.

Reply:
Proposition:

```